Dockerfile.alpine

```diff
@@ -0,0 +1,49 @@
+FROM golang:1.7.5-alpine
+MAINTAINER Gerry Gleason
+
+RUN apk add --update \
+      ca-certificates \
+      curl wget \
+      curl-dev \
+      procps \
+      openrc \
+      git \ 
+      make \
+    && rm -rf /var/cache/apk/* \
+    \
+    && addgroup holochain -g 868 \
+    && adduser -g 'Standard holochain user.' -u 868 -G holochain -h /home/holochain -s /bin/sh -D holochain \
+    && mkdir -p ~holochain/.ssh \
+    && chmod 700 ~holochain/.ssh \
+    && mv /etc/profile.d/color_prompt /etc/profile.d/color_prompt.sh \
+    && sed -i -e 's#/bin/ash#/bin/sh#' /etc/passwd \
+    && mkdir -p /work/bin /work/golang \
+    \
+    && touch /etc/holochain.env_vars \
+    && echo 'export GOPATH=/work/golang' >> /etc/holochain.env_vars \
+    && echo 'export GOBIN=/home/holochain/bin' >> /etc/holochain.env_vars \
+    && echo 'export PATH=$GOPATH/bin:/usr/local/go/bin:$GOBIN:$PATH' >> /etc/holochain.env_vars \
+    && cat /etc/holochain.env_vars \
+    && chown -R holochain /work \
+    && su holochain -c 'source /etc/holochain.env_vars; \
+      go get -v -u github.com/whyrusleeping/gx \
+      && go get -v -d github.com/metacurrency/holochain \
+      && cd /work/golang/src/github.com/metacurrency/holochain \
+      && make deps'
+
+WORKDIR /work/golang/src/github.com/metacurrency/holochain
+
+# above should get cached, does not depend on source
+ADD . /work/holochain 
+
+RUN rm -rf /work/golang/src/github.com/metacurrency/holochain/* \
+    && cp -pr /work/holochain/* /work/golang/src/github.com/metacurrency/holochain/ \
+    && chown -R holochain /work \
+    && su holochain -c 'source /etc/holochain.env_vars; pwd; ls -tlr; make; make bs'
+
+CMD ["su", "holochain", "-c", "source /etc/holochain.env_vars; make test" ]
```

Comment:
With this you build and test with:
docker build -t holochain:alpine -f Dockerfile.alpine .
docker run -t holochain:alpine
You can run an hc command in the container: docker exec -t holochain:alpine su holochain -c "source /etc/holochain.env_vars; hc <args>"
It will stay up as until hc exits.